bd-logger/src/log_replay.rs

```diff
@@ -254,11 +256,7 @@ impl ProcessingPipeline {
         );
       }
 
-      if result.take_screenshot {
-        if let Err(e) = self.take_screenshot_tx.try_send(()) {
-          log::debug!("failed to send take screenshot signal: {e}");
-        }
-      }
+      self.take_screenshot_handler.take_screenshot();
```

Comment:
Don't you still need to guard this with result.take_screenshot? Seems like there should be some test to catch this?